samples/python/whisper_speech_recognition/README.md

```diff
@@ -15,9 +15,18 @@ optimum-cli export openvino --trust-remote-code --model openai/whisper-base whis
 
 ## Prepare audio file
 
-Prepare audio file in wav format with sampling rate 16k Hz.
+You can prepare an audio file in WAV format with a sampling rate of 16k Hz using the `recorder.py` script. The script will record 5 seconds of audio from the microphone. 
```

Comment:
Please, add the same to C++ README. But the link to recorder.py should point to python folder.

Reply:
Sure

Reply:
Let's merge this one first? I will open another one for C++ side.

Reply:
You could have already done that. But ok, let it be in another PR.